keywords/LiteServiOS.py

```diff
@@ -131,8 +287,20 @@ def stop(self):
 
         liteserv_admin_url = "http://{}:59850".format(self.host)
         log_info("Stopping LiteServ: {}".format(liteserv_admin_url))
-        resp = self.session.put("{}/stop".format(liteserv_admin_url))
-        log_r(resp)
-        resp.raise_for_status()
+
+        count = 0
+        while count < MAX_RETRIES:
```

Comment:
Why are you retrying here? `liteserv.stop` should only be called if a LiteServ is running.